compconfig/compconfig/functions.py

```diff
@@ -30,9 +34,10 @@ def get_inputs(meta_params_dict):
     meta_params = MetaParams()
     meta_params.adjust(meta_params_dict)
     params = Specifications()
+    params.specification(serializable=True)
```

Comment:
This and another calls to params.specifications()` in other functions doesn't seem to be working to help me pass the tests.  Still getting error that json not serializable.

Reply:
The specification method doesn't change the `params` object. You have to use it like this:
```python
spec = params.specification(serializable=True, meta_data=True, **otherargs)
```